content/source/docs/enterprise/before-installing/index.html.md

```diff
@@ -5,31 +5,33 @@ page_title: "Pre-Install Checklist - Before Installing - Terraform Enterprise"
 
 # Terraform Enterprise Pre-Install Checklist
 
-Before installing Terraform Enterprise, you'll need to prepare an appropriate instance, supporting services, and relevant data files. Please make careful note of these requirements, as the installation may not be successful if these requirements are not met.
+Before installing Terraform Enterprise, you'll need to make several key architecture decisions and prepare some infrastructure and data files. Please make careful note of these requirements, as the installation may not be successful if these requirements are not met.
 
 ## Summary
 
 Prepare all of the following before installing:
 
-1. **Credentials:** Ensure you have a Terraform Enterprise license and a TLS certificate for Terraform Enterprise to use.
-2. **Operational mode decision:** Decide how Terraform Enterprise should store its data: in discrete external services, or on a plain mounted disk.
-3. **Data storage:** Depending on your operational mode, prepare data storage services or a block storage device.
-4. **Linux instance:** Prepare a Linux instance for Terraform Enterprise. This might require additional configuration or software installation, depending on the OS and your operational requirements.
+1. **Choose a deployment method:** Decide whether to perform a clustered deployment of Terraform Enterprise (using a HashiCorp-provided Terraform module) or use the installer to deploy individual instances.
+2. **Choose an operational mode:** Decide how Terraform Enterprise should store its data. This is affected by your choice of deployment method.
+3. **Credentials:** Ensure you have a Terraform Enterprise license and a TLS certificate for Terraform Enterprise to use.
+4. **Data storage:** Depending on your operational mode, prepare data storage services or a block storage device.
+5. **Linux instance:** Prepare a Linux machine image (if clustering) or a running Linux instance (if deploying individually) for Terraform Enterprise. This might require additional configuration or software installation, depending on the OS and your operational requirements.
```

Comment:
Should we point out preparing an image is optional? They can use a custom OS image, or use the cloud-provided ones we've got as default.

Reply:
Mm, good call.